test/utilities/oas-schema/oas-schema.test.ts

```diff
@@ -72,17 +62,87 @@ describe('OASSchema', () => {
             },
           },
         ],
+      };
+    });
+
+    it('calls the provided operation with the provided parameters without apiKey', async () => {
+      const operation = new OASOperation(operationObject);
+      const executeMock = jest.fn(
+        (_arg) => new Promise((resolve) => resolve(_arg)),
+      );
+      const filePath = 'test/fixtures/facilities_oas.json';
+      const schema = await generateSchema(filePath);
+
+      // eslint-disable-next-line dot-notation
```

Comment:
What does this do?

Reply:
Looks like it was left in by accident, good catch! ⚾